docs/repositories-configure/ignoring-files.md

```diff
@@ -10,16 +10,16 @@ To exclude files from your repository analysis open your repository **Settings**
 
 ![Ignored files](images/ignored-files.png)
 
-You can also ignore files using your own tool configuration files, although this depends on the option being supported by each tool.
+You can also ignore files using your own [tool configuration files](configuring-code-patterns.md#using-your-own-tool-configuration-files), although this depends on the option being supported by each tool.
 
-If you need more flexibility in ignoring files [use a Codacy configuration file](codacy-configuration-file.md) instead.
+If you need more flexibility in ignoring files, use a Codacy configuration file to [define a custom list of file paths to exclude](codacy-configuration-file.md#syntax-for-ignoring-files) instead.
```

Comment:
Here we can point to the specific section with the ignore syntax and add examples there.

Reply:
Agree.
Small tweak for clarity, if it makes sense to you.

```suggestion
If you need more flexibility in ignoring files, use instead a Codacy configuration file to [define a custom list of file paths to exclude](codacy-configuration-file.md#syntax-for-ignoring-files).
```

Reply:
Or we could remove it? 🤔 It sounds optional.

```suggestion
If you need more flexibility in ignoring files, use a Codacy configuration file to [define a custom list of file paths to exclude](codacy-configuration-file.md#syntax-for-ignoring-files).
```